azurerm/internal/services/purview/purview_account_resource.go

```diff
@@ -0,0 +1,290 @@
+			Update: schema.DefaultTimeout(30 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.AccountName(),
+			},
+
+			"resource_group_name": azure.SchemaResourceGroupName(),
+
+			"location": azure.SchemaLocation(),
+
+			"sku_capacity": {
+				Type:     schema.TypeInt,
+				Required: true,
+				ValidateFunc: validation.IntInSlice([]int{
+					4,
+					16,
+				}),
+			},
+
+			"public_network_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+
+			"identity": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"principal_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"tenant_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+
+			"catalog_endpoint": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"guardian_endpoint": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"scan_endpoint": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"atlas_kafka_endpoint_primary_connection_string": {
+				Type:      schema.TypeString,
+				Computed:  true,
+				Sensitive: true,
+			},
+
+			"atlas_kafka_endpoint_secondary_connection_string": {
+				Type:      schema.TypeString,
+				Computed:  true,
+				Sensitive: true,
+			},
+
+			"tags": tags.Schema(),
+		},
+	}
+}
+
+func resourcePurviewAccountCreateUpdate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*clients.Client).Purview.AccountsClient
+	subscriptionId := meta.(*clients.Client).Account.SubscriptionId
+	ctx, cancel := timeouts.ForCreateUpdate(meta.(*clients.Client).StopContext, d)
+	defer cancel()
+
+	name := d.Get("name").(string)
+	location := azure.NormalizeLocation(d.Get("location").(string))
+	resourceGroup := d.Get("resource_group_name").(string)
+	t := d.Get("tags").(map[string]interface{})
+
+	if d.IsNewResource() {
+		existing, err := client.Get(ctx, resourceGroup, name)
+		if err != nil {
+			if !utils.ResponseWasNotFound(existing.Response) {
+				return fmt.Errorf("Error checking for presence of existing Purview Account %q (Resource Group %q): %s", name, resourceGroup, err)
+			}
+		}
+
+		if existing.ID != nil && *existing.ID != "" {
+			return tf.ImportAsExistsError("azurerm_purview_account", *existing.ID)
+		}
+	}
+
+	account := purview.Account{
+		AccountProperties: &purview.AccountProperties{},
+		Identity: &purview.Identity{
+			Type: purview.SystemAssigned,
+		},
+		Location: &location,
+		Sku: &purview.AccountSku{
+			Capacity: utils.Int32(int32(d.Get("sku_capacity").(int))),
+			Name:     purview.Standard,
```

Comment:
is there a reason we are not exposing the sku as a property?

Reply:
Dunno if I'm off-base here, but I assume they will introduce more, can we expose complete property with the only accepted value of `Standard` for the name? Would save time when it's exposed later on for the future generations wanting to expand this one?

Reply:
That makes sense. I will expose this property.